CMakeLists.txt

```diff
@@ -162,7 +174,7 @@ function(generate_version_gen_c)
 
   add_custom_target(
     create_version_gen ALL
-    COMMAND ${CMAKE_SOURCE_DIR}/tools/create_version_c.sh ${CMAKE_BINARY_DIR}/src/version_gen.c ${_VERSION}
+    COMMAND ${CMAKE_SOURCE_DIR}/tools/create_version_c.sh ${DARKTABLE_BINDIR}/version_gen.c ${_VERSION}
```

Comment:
Source files don't belong in the `bin/` dir.

Reply:
It's a generated file, so not real source code. It's also not installed but placed in the build tree, where intermediate files belong.

Reply:
btw, I did not change the logic to how it was before. The only change is that the old `src` is now `bin`, which was necessary to have a structure in the build-tree like in the install-tree.

old: CMAKE_BINARY_DIR/src: xxx/darktable.git/build/src,
new: DARKTABLE_BINDIR: xxx/darktable.git/build/bin